### ISSUE TITLE
Request for an addition to the documentation - plugin is not compatible with... And

### ISSUE THREAD

**Issue by EdmundsEcho** [Yggdroot/indentLine#231]
First of all, thank you for this plugin that greatly improves my workflow.  I have a tall screen; the visual queue helps a lot.

While obvious in hindsight, it took me a while to figure out why my Haskell operators were barely visible.  What I want to be subtle in indentLine needs to be visually distinctive in `vim-haskellConcealPlus` (and similar plugins).  It would be _great_ if you had a list of plugins that also rely on the `conceal` setting and `highlight group`.  

The AND in the title: Is there a way to create a separate `highlight group` for the visual markers the `indentLine` inserts into the buffer?  Right now it uses `Conceal`.  If the name were `ConcealIndentLine`, I could use both plugins.  As of now, I have to disable it in my Haskell work where the visual queue for alignment is less a priority when compared to the code proper.

**Comment by Yggdroot** [comment]
First, I don't know what plugins have used `conceal`.
Second, there is only one way to customize the color, i.e., `hi Conceal`.

**Comment by EdmundsEcho** [comment]
Good point.  How about collecting a list from users, and/or giving examples of a few plugins that might conflict?  Generally, it's a plugin that relies on `Conceal`.  There is a perfect setting to exclude certain files types and thus provide a way to help us figure out which ones to exclude e.g., `json` if you want to see quotes, `haskellConcealPlus` to show -> as a single looking char without changing the underlying code.

**Comment by ahmedelgabri** [comment]
> Second, there is only one way to customize the color, i.e., `hi Conceal`.

@Yggdroot Any reason why is `hi Conceal` the only way & not create a custom group for the plugin only as @EdmundsEcho something like `hi ConcealIndentLine` or `hi IndentLine`?

Because I'm also having the same issue but with https://github.com/pangloss/vim-javascript#concealing-characters I like to conceal the `function` keyword.

**Comment by Yggdroot** [comment]
@ahmedelgabri It's vim's ability.

**Comment by ahmedelgabri** [comment]
@Yggdroot that's because the plugin depends on `conceal` right? is there is anyway else?

**Comment by Yggdroot** [comment]
yes. no.

**Comment by ferrwan** [comment]
Having same issue with [vim-javascript's conceal](https://github.com/pangloss/vim-javascript#concealing-characters). vim-indentline override vim-javascript's conceal, but if I use [this version](https://github.com/Yggdroot/indentLine/commit/0a6171444fe036411c3155c04d4022a7b4af4bb6) (it's the version before @Yggdroot refactor the code) they both went well.

**Comment by Saul-Mirone** [comment]
same as @ferrwan

**Comment by ygrek** [comment]
ref https://github.com/Yggdroot/indentLine/issues/155
